### DISCUSSION TITLE
How to use MySQL as database not noly sqlite.

### DISCUSSION THREAD

**Discussion by liuzhaobing** [open-webui/open-webui#4544] [Q&A]
I'd like to using MySQL to store our data rather than sqlite, how should I config?

**Comment by zspine** [comment]
@liuzhaobing I don't believe MySQL is supported yet, but you can use PostgreSQL instead by configuring the DATABASE_URL environment variable. I'm already using this setup without any issues.

Here's an example of how to set it up:

```
DATABASE_URL=postgres://postgres_user:postgres_pass@postgres:5432/postgres_db
```

You can find more information in the documentation:
https://docs.openwebui.com/features#-integration--security
